### PR TITLE
Print reason for re-running resolver in verbose mode

### PR DESCRIPTION
**Pull request by hartbit**

### CODE REVIEW

File: Sources/Commands/SwiftTool.swift [hunk 1]
```diff
@@ -463,11 +504,11 @@ public class SwiftTool<Options: ToolOptions> {
     private var _workspace: Workspace?
 
     /// Returns the currently active workspace.
-    func getActiveWorkspace() throws -> Workspace {
+    func getActiveWorkspace(options: Options) throws -> Workspace {
```
Comment (aciidb0mb3r) [inline Sources/Commands/SwiftTool.swift]: SwiftTool class already has access to options so you don't need to pass it in.

**Comment by aciidb0mb3r** [comment]
Looks great! I just have one comment.

**Comment by hartbit** [comment]
I finished this PR and based it off #2424 to be able to get advantage that packages always have names.

**Comment by hartbit** [comment]
@swift-ci please smoke test

**Comment by hartbit** [comment]
@swift-ci please smoke test

**Pull request merged**